src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -258,6 +264,23 @@ export default App.extend({
 
     Radio.request('alert', 'show:success', renderTemplate(BulkEditActionsSuccessTemplate, { itemCount }));
   },
+  showCountView() {
+    if (!this.collection || !this.filteredCollection.length) {
```

Comment:
you know what else would work here?  Since the `CountView` is pretty simple if could do this `!this.collection || !this.filteredCollection.length` and then just render an empty template?  I think a `<div></div>` isn't going to hurt anything?